relayer/processor/path_end.go

```diff
@@ -3,6 +3,7 @@ package processor
 // PathEnd references one chain involved in a path.
 // A path is composed of two PathEnds.
 type PathEnd struct {
+	Path     string
```

Comment:
Is `Path` descriptive enough? Should it be `PathName` or should there be a comment on the field?

Reply:
changed to `PathName`